pyproject.toml

```diff
@@ -10,6 +10,7 @@ readme.content-type = "text/markdown"
 keywords = ["virtual", "environments", "isolated", "testing"]
 license = "MIT"
 urls.Homepage = "http://tox.readthedocs.org"
+url.Documentation = "https://tox.wiki/"
```

Comment:
I am not super familiar with pyproject.toml to configure meta data for projects, but I am pretty sure this should be `urls.Documentation`, right?

See https://hatch.pypa.io/latest/config/metadata/#urls

Reply:
how many errors can I put in a one-line change...

thanks! Fixed.